docs/src/main/sphinx/admin/fault-tolerant-execution.rst

```diff
@@ -36,28 +36,23 @@ depending on the desired :ref:`retry policy <fte-retry-policy>`.
 
 .. warning::
 
-  Setting ``retry-policy`` disables :ref:`write operations
-  <sql-write-operations>` with connectors that do not support fault-tolerant
-  execution of write operations, resulting in a "This connector does not support
-  query retries" error message.
+  Setting ``retry-policy`` may cause queries to fail with connectors that do not
+  explicitly support fault-tolerant execution, resulting in a "This connector
+  does not support query retries" error message.
 
   Support for fault-tolerant execution of SQL statements varies on a
-  per-connector basis:
-
-  * Fault-tolerant execution of :ref:`read operations <sql-read-operations>` is
-    supported by all connectors.
-  * Fault-tolerant execution of :ref:`write operations <sql-write-operations>`
-    is supported by the following connectors:
-
-    * :doc:`/connector/bigquery`
-    * :doc:`/connector/delta-lake`
-    * :doc:`/connector/hive`
-    * :doc:`/connector/iceberg`
-    * :doc:`/connector/mongodb`
-    * :doc:`/connector/mysql`
-    * :doc:`/connector/postgresql`
-    * :doc:`/connector/redshift`
-    * :doc:`/connector/sqlserver`
+  per-connector basis, with more details in the documentation for each
+  connector. The following connectors support fault-tolerant execution:
```

Comment:
This should probably clarify that "support FTE" here means supports "some form of FTE" (i.e. only QUERY retries during READS is also sufficient to be included in this list).